spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'rails_helper'
+
+RSpec.describe OpenidConnectAuthorizeForm do
+  subject(:form) do
+    OpenidConnectAuthorizeForm.new(
+      acr_values: acr_values,
+      client_id: client_id,
+      nonce: nonce,
+      prompt: prompt,
+      redirect_uri: redirect_uri,
+      response_type: response_type,
+      scope: scope,
+      state: state
+    )
+  end
+
+  let(:acr_values) do
+    'http://idmanagement.gov/ns/assurance/loa/1 http://idmanagement.gov/ns/assurance/loa/3'
+  end
```

Comment:
I wanted to make sure it was clear it was a space-separated string, but I'll update to use constants